Backend/src/main/java/com/fmi/cinema/cinema/cinema/model/Movie.java

```diff
@@ -1,8 +1,8 @@
-package com.fmi.cinema.cinema.movies.model;
+package com.fmi.cinema.cinema.cinema.model;
```

Comment:
cinema.cinema.cinema? no need just make it cinema.cinema.XXXX there is no need for an additional cinema package